src/components/form/form.style.js

```diff
@@ -53,6 +54,15 @@ export const StyledForm = styled.form`
       theme.spacing * fieldSpacing}px;
   }
 
+  ${StyledInlineInputs} {
```

Comment:
How about adding another `&` to raise specificity of the rule on line 51 instead of adding this whole block?

Reply:
there's no need to do this, as it will specifically target inline inputs, I don't understand why should I raise specificity of it.